crates/pera-types/src/messages_signature_mpc.rs

```diff
@@ -0,0 +1,50 @@
+use crate::committee::EpochId;
+use crate::crypto::{default_hash, AuthoritySignInfo};
```

Comment:
From a structure POV, would it not be better to structure these files as :
- `messages.rs`, and
- `messages/signature_mpc.rs`,
- `messages/consensus.rs`
?

Reply:
Not crucial atm imo. Anyway such refactor will cause many conflicts in the following prs, so I'll rather create an issue. Wdyt?

Reply:
good point.